### ISSUE TITLE
Tenant identification

### ISSUE THREAD

**Issue by orderkingegypt** [Alhumsiabdo/multy_tenancy_order_king#1]
Hello,

can you please send me your email?

And your project is missing something important.
We have many online-shops with different domains.

The tenant identification has to be done via domain.

Currently you're only identifying your tenants by email and role.

Best regards
Maciej

**Comment by Alhumsiabdo** [comment]
The assignment you requested from me is not present in the file that was sent. Do you mean that I should create a table of user domains and make it a primary key across the relationship? If this is what you mean, then I believe this assignment is unnecessary. I apologize for the delayed response. I am currently in the process of moving to a new apartment, which has caused the delay. Regarding email, it appears to be working fine. I believe the issue may be on your side

**Comment by orderkingegypt** [comment]
Hello,
the main tenant (merchant or online-shop owner) identification should be done by domain. Then when the tenant is identified by the domain a second "role" identification has to take place (admin, merchant, user)

My question to you is, how would you achieve the tenant identification by domain and how would you get the users from the tenant?

**Comment by Alhumsiabdo** [comment]
I can achieve this using an Nginx server because I can configure Nginx files to create sub domains and link them to specific database paths. Nginx allows me to set up multiple domains for the same project

**Comment by orderkingegypt** [comment]
![WhatsApp Image 2023-10-23 at 17 55 15](https://github.com/Alhumsiabdo/multy_tenancy_order_king/assets/145200823/ef29dc96-9495-4a69-bd27-a0d9e0ca3c02)

**Comment by Alhumsiabdo** [comment]
yes i understand what do you mean now, I can make that by creating that by create table name tenats table columns id ,name, domain and also I can create a custom Middleware to handle this
and i will define route like this [Route::domain('{tenant}.yourapp.com')->group(function ($tenant) {
    // Define routes for this tenant
});]  and i also can male Middleware custom for this
